Formula/vagrant-completion.rb

```diff
@@ -4,12 +4,15 @@ class VagrantCompletion < Formula
   url "https://github.com/hashicorp/vagrant/archive/v2.2.16.tar.gz"
   sha256 "ab3c60bb12b2da916fd073192849f2b5d3f224f95febf3538212247c4cde28d6"
   license "MIT"
+  revision 1
   head "https://github.com/hashicorp/vagrant.git"
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "325bfccadee57150fad636b0ed58536dc47659556fbf34c9ca8ac88dde686e7d"
   end
 
+  conflicts_with "zsh-completions", because: "the zsh-completions formula includes zsh completion"
```

Comment:
A similar line should be added to `zsh-completions` formula as well

Reply:
Thanks, @bayandin, sent a  PR #79305